energy_datetime/calendar.py

```diff
@@ -1,3 +1,6 @@
+"""
+contains the business logic (namely the BDEW calendar information, mainly based on Germany holiday calendar)

```

Comment:
Reduce module description to avoid pylint complaint:

> energy_datetime/calendar.py:2:0: C0301: Line too long (108/100) (line-too-long)

```suggestion
contains the business logic (namely the BDEW calendar information)

```